tests/fmt.toml

```diff
@@ -2,7 +2,6 @@ comment_width=100
 wrap_comments=true
 format_code_in_doc_comments=true
 format_strings=true
-license_template_path=./tests/license.txt
```

Comment:
Why that?

Reply:
It is no longer supported.

Reply:
A comment in the commit message would be helpful